app/__init__.py

```diff
@@ -55,6 +56,7 @@ def apply_driver_hacks(self, app, info, options):
 aws_ses_client = AwsSesClient()
 aws_ses_stub_client = AwsSesStubClient()
 aws_sns_client = AwsSnsClient()
+aws_cloudwatch_client = AwsCloudwatchClient()
```

Comment:
[suggestion] we should either configure (call `init_app`) this client in this file alongside the ses and sns clients, and then use this instance, or remove the client from this file and continue to initialize it within `provider_tasks` as you currently have it